frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx

```diff
@@ -33,11 +35,23 @@ export const CycleTime = ({ title }: cycleTimeProps) => {
           : item
       )
     )
-    dispatch(saveDoneColumn([]))
+
+    if (value === DONE) {
+      setSaveDone([...saveDone, name])
+      dispatch(saveDoneColumn([]))
+    } else if (saveDone.includes(name)) {
+      setSaveDone(saveDone.filter((e) => e !== name))
+      dispatch(saveDoneColumn([]))
+    }
   }
 
   useEffect(() => {
     dispatch(saveCycleTimeSettings(cycleTimeOptions))
+    cycleTimeOptions.forEach((item) => {
+      if (item.value === 'Done') {
```

Comment:
seems we have defined the constant 'Done' in constants.ts, probably let's use the constant here as well to maintain consistency.

Reply:
Fixed, thanks ziwen!